docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/client-side-validation.md

```diff
@@ -23,39 +23,37 @@ The implementation of client-side validation depends on what kind of client appl
 
 ### Validation in Xamarin mobile apps
 
--   **Validate Text Input and Show Errors**
+-   **Validate Text Input and Show Errors** <br/>
```

Comment:
Hi @BillWagner,

Had been working on the Docker lifecycle book, resuming here now.

Regarding this issue I found in one of the documents that backslash `\` gets the same result.

I had been using the two spaces, although I didn't like them, because they're not obvious, then @mairaw suggested to stop using them, so I opted for the `<br/>`.

Any way I've been changing all to '\' because it's pretty clear, the outcome looks good and it's plain markdown. 😊